prime-dotnet-webapi/Extensions/ClaimsPrincipalExtensions.cs

```diff
@@ -32,6 +32,20 @@ public static bool CanEdit(this ClaimsPrincipal User, Enrollee enrollee)
             return !PrimeUserId.Equals(Guid.Empty) && PrimeUserId.Equals(enrollee.UserId);
         }
 
+        /// <summary>
+        /// Returns true if the logged in user is an admin, or if the user has the same UserId as the record
+        /// </summary>
+        public static bool PartyCanEdit(this ClaimsPrincipal User, Party party)
```

Comment:
I feel like CanEdit could be updated to take in an interface object for both Enrollee and Party. Something like
`public interface IUserBoundModel
    {
        Guid UserId { get; set; }
    }`